vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/test/java/com/vaadin/flow/component/datepicker/DatePickerVariantTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2000-2021 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datepicker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class DatePickerVariantTest {
+
+    private final DatePicker datePicker = new DatePicker();
+
+    @Test
+    public void addAndRemoveLumoAlignCenterVariant_themeAttributeUpdated() {
+        assertThemeAttribute(null);
+        datePicker.addThemeVariants(DatePickerVariant.LUMO_ALIGN_CENTER);
+        assertThemeAttribute("align-center");
+        datePicker.removeThemeVariants(DatePickerVariant.LUMO_ALIGN_CENTER);
```

Comment:
Similarly removing the theme variants in every test is not necessary then.

We should rather have these additional tests:
- verify that all theme variants can be removed
- add multiple variants, and verify that a single theme variant can be removed, while the others are kept in place

I would also suggest to add a test to verify that calling `addThemeVariants` multiple times does indeed concatenate them, instead of overwriting the results of previous calls.